qcodes/instrument/channel.py

```diff
@@ -787,30 +789,31 @@ class AutoLoadableChannelList(ChannelList):
             ways and should not be repeated in an instrument snapshot.
 
         multichan_paramclass: The class of
-            the object to be returned by the :class:`ChannelList` ``__getattr__`` method.
+            the object to be returned by the
+            :class:`ChannelList` ``__getattr__`` method.
             Should be a subclass of :class:`MultiChannelInstrumentParameter`.
 
         **kwargs: Keyword arguments to be passed to the ``load_from_instrument``
             method of the channel class. Note that the kwargs are *NOT* passed
             to the ``__init__`` of the super class.
 
     Raises:
-        ValueError: If :class:`chan_type` is not a subclass of :class:`InstrumentChannel`
+        ValueError: If :class:`chan_type` is not a subclass of
+            :class:`InstrumentChannel`
         ValueError: If ``multichan_paramclass`` is not a subclass of
-            :class:`MultiChannelInstrumentParameter` (note that a class is a subclass
-            of itself).
-
+            :class:`MultiChannelInstrumentParameter` (note that a class is a
+            subclass of itself).
     """
     def __init__(
```

Comment:
I am not a big fan of this indention style but left it for now

Reply:
what would you recommend for the case of many arguments each having long type annotations?

Reply:
One per line is fine but this ( on separate lines thing is not great. These lines are actually already too long even with this style